debugtools/DDR_VM/src/com/ibm/j9ddr/tools/PointerGenerator.java

```diff
@@ -876,12 +884,23 @@ private String getPointerType(String type) {
 		return type.toUpperCase().substring(0, 1) + type.substring(1) + "Pointer";
 	}
 
+	private int errorCount = 0;
+
 	private void writeArrayMethod(PrintWriter writer, StructureDescriptor structure, FieldDescriptor fieldDescriptor) {
-		String returnType = getArrayType(fieldDescriptor.getType());
-		if (returnType.equals("EnumPointer")) {
-			writeEnumEAMethod(writer, returnType, structure, fieldDescriptor);
-		} else {
-			writeEAMethod(writer, returnType, structure, fieldDescriptor);
+		try {
+			String returnType = getArrayType(fieldDescriptor.getType());
+			if (returnType.equals("EnumPointer")) {
+				writeEnumEAMethod(writer, returnType, structure, fieldDescriptor);
+			} else {
+				writeEAMethod(writer, returnType, structure, fieldDescriptor);
+			}
+		} catch (Exception e) {
+			errorCount += 1;
+			if (errorCount < 100) {
```

Comment:
Please add a comment to explain the purpose of throwing up to 100 errors.

Reply:
Done.